tailwind.config.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
           "base-100": "#ffffff",
           "base-200": "#f1f5f9",
           "base-300": "#e2e8f0",
+          "base-400": "#5f6368",
```

Comment:
Ah... I see you've added this. You can disregard my other comment about it not being valid.